### ISSUE TITLE
Unable to install Anaconda

### ISSUE THREAD

**Issue by agrawalmanisha** [ContinuumIO/anaconda-issues#1763]
Dear Sir,

After the previous problem, I have formatted my system and trying to reinstall Anaconda but I am getting a lot of errors as follows:

1. Error Opening file for writing: C:\User\Python\preconda.tar.pz2
I am being asked to either abort the installation or ignore. After ignoring I am getting the foll error:

2. Error opening file for writing : C:\User\Python\Lib\_nsis.py

Thus during installation I am not being able to install any of the preconda.tar.pz2 , nsis.py , _system_path.py , python-3.6.0-0.tar.bz2 , all other tar.bz files. 

So basically I am not being able to install Anaconda. Please help. I am using Windows 10.

Regards,
Manisha Agrawal

**Comment by PRASAD-406** [comment]
> Dear Sir,
> 
> After the previous problem, I have formatted my system and trying to reinstall Anaconda but I am getting a lot of errors as follows:
> 
> 1. Error Opening file for writing: C:\User\Python\preconda.tar.pz2
>    I am being asked to either abort the installation or ignore. After ignoring I am getting the foll error:
> 2. Error opening file for writing : C:\User\Python\Lib_nsis.py
> 
> Thus during installation I am not being able to install any of the preconda.tar.pz2 , nsis.py , _system_path.py , python-3.6.0-0.tar.bz2 , all other tar.bz files.
> 
> So basically I am not being able to install Anaconda. Please help. I am using Windows 10.
> 
> Regards,
> Manisha Agrawal

Hello Manisha,

Have this issue is solved ? Because i'm facing the same error now
